### PR TITLE
chore(v3): refine error handling

### PR DESCRIPTION
**Pull request by hund030**

### CODE REVIEW

**Review by jayzhang — APPROVED** [review]

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -887,5 +887,6 @@
   "depChecker.linuxDepsNotFound": "Cannot find @SupportedPackages. Install @SupportedPackages manually and restart Visual Studio Code.",
   "depChecker.linuxDepsNotFoundHelpLinkMessage": "Cannot find @SupportedPackages.\n\nTeams Toolkit requires these dependencies.",
   "error.ActionNotExist": "Action not found: %s",
-  "error.ComponentNotExist": "Component not found: %s"
+  "error.ComponentNotExist": "Component not found: %s",
+  "error.BadComponent": "Component '%s' lacks of property: %s"
```
Comment (IvanJobs) [inline packages/fx-core/resource/package.nls.json:891]: Nit: how about adding `.` in the end to make it into a complete sentence?

Comment (IvanJobs) [inline packages/fx-core/resource/package.nls.json:891]: Nit: how about adding `.` in the end to make it into a complete sentence?

Comment (hund030) [reply packages/fx-core/resource/package.nls.json:891]: Good catch. Will fix in next PR.

File: packages/fx-core/src/component/error.ts [hunk 2]
```diff
@@ -20,3 +21,86 @@ export class ComponentNotExist extends SystemError {
...
+    );
+  }
+}
+
+export class UnzipError extends UserError {
+  constructor(source: string, path?: string) {
+    super(
+      source,
+      new.target.name,
+      `Failed to unzip templates and write to disk. Suggestions: ${[
```
Comment (IvanJobs) [inline packages/fx-core/src/component/error.ts:74]: How about localization of the error messages?

**Review by IvanJobs — APPROVED** [review]
lgtm for bot

**Pull request merged**